ui/webui/src/components/AnacondaWizard.jsx

```diff
@@ -75,12 +75,6 @@ export const AnacondaWizard = ({ dispatch, storageData, localizationData, onAddE
     const isBootIso = conf["Installation System"].type === "BOOT_ISO";
 
     const stepsOrder = [
-        {
```

Comment:
I would prefer inline. unshift works fine but the changed order makes this harder to read.

See here how to conditionally have an item in an array.
```
const arr = [
  ...(cond ? ['a'] : []),
  'b',
];
```

Reply:
I tried to do it this way too, but any empty value causes inconsistency in the UI. So I find another way to do this.